qualtran/bloqs/mod_arithmetic/_shims.py

```diff
@@ -103,6 +117,29 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> 'BloqCountDictT':
             Swap(self.n): 1,
         }
 
+    def on_classical_vals(
```

Comment:
why use the shim instead of the actual implementation?

Reply:
I only implemented enough to test that ECAdd works - when ModInverse bloq is merged I'm happy to replace the shim with that in the construction/tests.